### PR TITLE
Misc improvements of ops tooling

### PR DESCRIPTION
**Pull request by d10r**
* support for EIP-1559 gas price settings in truffle config
* default EIP-1559 gas price settings for Polygon in order to avoid "underpriced" failures
* auto-detection of gov owner type (multisig or EOA)
* allow to skip tokens in `info-show-protocol.js`
* fix duplicate removal of forwarders printing

### CODE REVIEW

File: packages/ethereum-contracts/truffle-config.js [hunk 1]
```diff
@@ -123,6 +125,8 @@ function createNetworkDefaultConfiguration(
                 shareNonce: true,
             }),
         gasPrice: +getEnvValue(networkName, "GAS_PRICE"),
+        maxFeePerGas: +getEnvValue(networkName, "MAX_FEE_PER_GAS"),
```
Comment (0xdavinchee) [inline packages/ethereum-contracts/truffle-config.js:128]: Do we need/want to include `maxFeePerGas` and `maxPriorityFeePerGas` in configs for networks (or maybe in the env for CI workflows) other than matic to enable type 2 transactions?

**Review by 0xdavinchee — APPROVED** [review]
Everything looks great! 
Just had one small question, which I just wanted to double check out of curiosity.

File: packages/ethereum-contracts/contracts/interfaces/utils/IMultiSigWallet.sol [hunk 2]
```diff
@@ -9,4 +9,9 @@ interface IMultiSigWallet {
     function submitTransaction(address destination, uint value, bytes calldata data)
         external
         returns (uint transactionId);
+
+    // used for interface probing
+    function required()
```
Comment (hellwolf) [inline packages/ethereum-contracts/contracts/interfaces/utils/IMultiSigWallet.sol:15]: Add a reference in the contract's comment: https://github.com/gnosis/MultiSigWallet/blob/master/contracts/MultiSigWalletWithDailyLimit.sol

File: packages/ethereum-contracts/scripts/libs/common.js [hunk 3]
```diff
@@ -210,13 +217,31 @@ async function sendGovernanceAction(sf, actionFn) {
...
+        default: {
+            throw new Error("No known admin type specified and autodetect failed");
+        }
+    }
+}
+
+// Probes the given account to see what kind of admin it is.
+// Possible return values: "MULTISIG", "OWNABLE".
+// Throws when encountering an unknown contract.
+async function autodetectGovAdminType(sf, account) {
```
Comment (hellwolf) [inline packages/ethereum-contracts/scripts/libs/common.js:236]: Also need to remove github workflow "required" parameter for enabling this feature there.

**Review by hellwolf — APPROVED** [review]

**Comment by hellwolf** [comment]
Let's go? Please let's make the commit message legible too.

**Pull request merged**